.changeset/quick-balloons-drum.md

```diff
@@ -0,0 +1,6 @@
+---
+'@itwin/itwinui-react': major
+'@itwin/itwinui-css': major
```

Comment:
I think you want to remove this (added css changeset in other file)

Reply:
changed